internal/db/messages.go

```diff
@@ -22,7 +22,9 @@ import (
 )
 
 func (db *db) handleMessages(ctx context.Context, sub *event.Subscription) {
-	queue := newMergeQueue()
+	docIdQueue := newMergeQueue()
```

Comment:
suggestion: `docIDQueue` would be more idomatic.

Reply:
2 years later and I'm still messing that up :grin:

- [ ] docIDQueue